huxley/api/tests/test_delegate.py

```diff
@@ -200,3 +202,283 @@ def test_superuser(self):
             "email" : unicode(self.params['email']),
             "summary" : unicode(self.params['summary']),}
         )
+
+
+class DelegateListGetTestCase(tests.ListAPITestCase):
+    url_name = 'api:delegate_list'
+
+    def setUp(self):
+        self.user = models.new_user(username='regular', password='user')
+        self.school = models.new_school(user=self.user)
+        self.country = models.new_country()
```

Comment:
Don't think we need to explicitly construct the country or the committees, we don't care about them in this test